tests/integration/test_end_to_end_dcp.py

```diff
@@ -118,6 +117,9 @@ def test_smartseq2_run(self):
             except:
                 pass
 
+        print()
```

Comment:
empty print?

Reply:
This makes GitLab log output prettier (it was squished into another line for some reason).